check-setup-mds.log

```diff
@@ -0,0 +1,131 @@
+# MDS setup check 1.0.0
+
+## Operating system
+ProductName:	macOS
+ProductVersion:	11.4
+BuildVersion:	20F71
+
+MISSING You need macOS Catalina (10.15.x).
+
+## System programs
+OK        psql (PostgreSQL) 13.3
+OK        rstudio 1.4.1725
+OK        R 4.1.0 (2021-05-18) -- "Camp Pontanezen"
+OK        python 3.8.10
+OK        conda 4.10.3
+OK        bash 3.2.57(1)-release (x86_64-apple-darwin20)
+OK        git 2.30.1 (Apple Git-130)
+OK        make 3.81
+OK        latex 3.141592653-2.6-1.40.23 (TeX Live 2022/dev)
+OK        tlmgr 59291 (2021-05-21 05:14:40 +0200)
+OK        docker 20.10.7, build f0df350
+OK        code 1.58.0
+
+## Python packages
+OK        jupyterlab=3.0.16
+OK        numpy=1.21.0
+OK        pandas=1.2.5
+OK        flake8=3.9.2
+OK        black=21.5b2
+OK        nodejs=15.14.0
+OK        jupytext=1.11.3
+OK        jupyterlab-git=0.30.1
+MISSING   jupyterlab PDF-generation failed. Check that latex and jupyterlab are marked OK above.
```

Comment:
Why did this fail for you? Can you try running the command from the script (`jupyter nbconvert mds-nbconvert-pdf-test.ipynb --to pdf`) on an empty notebook that you create and let me know what goes wrong? Also a note to self to store the error message of that command via the script and only redirect stdout (and maybe also clean up the PDF to avoid confusion)

Reply:
I ran `jupyter nbconvert mds-nbconvert-pdf-test.ipynb --to pdf`, it complains about a latex style file `pdfcol.sty`:

`! LaTeX Error: File 'pdfcol.sty' not found.`

I tried the solution [here](https://github.com/jupyter/nbconvert/issues/1445#issuecomment-706632181) and it worked. Now I get all OKs in the log of `check-setup-mds.sh`. Should I add this to the instructions?

Reply:
@armanawn Yes can you add this as a "Note" section in a separate PR? I will merge this one now.

Reply:
Thanks @joelostblom, I will do that in an upcoming PR.

Reply:
This is the same as https://github.com/UBC-MDS/UBC-MDS.github.io/issues/102 which I will address in a PR soon.